frontend/test/metabase/containers/SaveQuestionModal.unit.spec.js

```diff
@@ -187,8 +217,8 @@ describe("SaveQuestionModal", () => {
       await waitFor(() => {
         expect(onCreateMock).toHaveBeenCalledTimes(1);
       });
-      expect(onCreateMock).toHaveBeenCalledWith({
-        ...question.card(),
+      checkCall(onCreateMock, {
```

Comment:
I find this new abstraction hard to follow, and it would be great to simplify this. E.g.
```
const newQuestion = onCreateMock.mock.calls[0];
expect(newQuestion.displayName()).toEqual(EXPECTED_SUGGESTED_NAME);
expect(newQuestion.description()).toBeNull();
expect(newQuestion.collectionId()).toBeNull();
```

Reply:
Yeah, we don't add functions for asserting on stuff. Eslint would also complaint about not having any assertions in a test if there's no `expect` in the test body and only `checkCall`

Reply:
I've factored this back into each test.